app/templates/tasklist.html

```diff
@@ -9,13 +9,26 @@ <h1 class="govuk-heading-xl">
       Round: {{application_meta_data["round"]|snake_case_to_human}}
     </h1>
     {% if not application_response["status"] == application_meta_data["submitted_status"] %}
-    <h2 class="govuk-body govuk-!-font-weight-bold">
-      {% if application_meta_data["number_of_completed_sections"] == application_meta_data["number_of_sections"] %}
-       All sections complete
-      {% else %}
-       In progress
+
+    <p class="govuk-body-l govuk-!-font-weight-bold">
+      Status: 
+      {% set completed_count = application_meta_data["number_of_completed_sections"] %}
+      {% set section_count = application_meta_data["number_of_sections"] %}
+      {% if completed_count == 0 %}
+      <strong class="govuk-tag govuk-tag--grey">
+        Not Started
+      </strong>
+      {% elif completed_count > 0 and completed_count < section_count %}
+      <strong class="govuk-tag govuk-tag--blue">
+        In Progress
+      </strong>
+      {% elif completed_count == section_count %}
+      <strong class="govuk-tag govuk-tag--green">
+        Complete
+      </strong>
       {% endif %}
-    </h2>
+    </p>
+    
     <p class="govuk-body govuk-!-margin-bottom-7">
```

Comment:
This seems unrelated, was it meant to be included?

Reply:
I  didn't change this file - just done a rebase before I merged